build.ts

```diff
@@ -0,0 +1,11 @@
+Bun.build({
+  entrypoints: ["src/index.ts"],
+  outdir: "dist",
+  /** @link https://github.com/oven-sh/bun/issues/159 */
+  format: "esm",
```

Comment:
CJS build is yet missing in this branch

Reply:
kinda fixed by [a381905](https://github.com/RobinTail/merge-sx/pull/443/commits/a3819052fc3eaa686d86bcd9f06670e758d658b7)

But still would be better to have the supported format by bun builder